windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,85 @@
+# Certbot for Window (beta)
+
+The Certbot development team is proud to propose you the first beta release of Certbot for Windows.
+
+This document explains how to install Certbot and use it on Windows.
+
+## Supported features
+
+All usual operations to create and manage an account, manage existing certificates, or select the ACME server, are supported.
+
+In terms of authenticator plugins, `standalone`, `manual` and `webroot` are supported. DNS plugins will be available soon.
+
+For now, no installer plugins are available. `apache`, `nginx` will be available soon, and a new installer for IIS is under development.
+
+Certificate automated renewals are supported.
+
+All options available for a given supported command will work as described by the documentation in Certbot for Windows.
+
+## Installation
+
+The following operations needs to be executed using an account with **administrative privileges** on the machine you want to install Certbot.
+
+Certbot can be installed through an installer that is available:
+- on GitHub: go to the Certbot GitHub release (https://github.com/certbot/certbot/releases), select the last version of Certbot (v0.40.0 currently), and download the installer named `certbot-beta-installer-win32.exe in the assets.
+- on Certbot official website: _[specific URL and procedure to be defined]_
+
+The Certbot development team ensures the authenticity of the Windows installer only for these two sources.
+
+Once downloaded, double-click on the installer to run it. You will need to select the location for Certbot installation directory, or keep the default one (`C:\Program Files(x86)\Certbot).
+
+At this point Certbot is ready to run.
+
+## Using Certbot
+
+Certbot is a command line interface with no UI. To interact with it you will need a command line: the classic one (`cmd.exe`) and Powershell aor both supported. You can also use Bash for Windows (available if you installed GIT for Windows for instance) if you prefer.
+
+The command line needs to be executed under **administrative privileges**. Otherwise Certbot will fail to run.
```

Comment:
I would say "with administrative privileges" or "under the administrator account".